platform/windows/crash_handler_windows.cpp

```diff
@@ -128,18 +129,25 @@ DWORD CrashHandlerException(EXCEPTION_POINTERS *ep) {
 	}
 
 	String msg;
+	String log_path;
 	const ProjectSettings *proj_settings = ProjectSettings::get_singleton();
 	if (proj_settings) {
-		msg = proj_settings->get("debug/settings/crash_handler/message");
+		msg = proj_settings->get_setting_with_override("debug/settings/crash_handler/message");
+		if (proj_settings->get_setting_with_override("debug/file_logging/enable_file_logging")) {
+			log_path = proj_settings->globalize_path(proj_settings->get_setting_with_override("debug/file_logging/log_path"));
+		}
 	}
 
 	// Tell MainLoop about the crash. This can be handled by users too in Node.
 	if (OS::get_singleton()->get_main_loop()) {
 		OS::get_singleton()->get_main_loop()->notification(MainLoop::NOTIFICATION_CRASH);
 	}
 
+	// Set window title while dumping the backtrace, as this can take 10+ seconds to finish.
+	DisplayServer::get_singleton()->window_set_title("ERROR: Program crashed, dumping backtrace to log...");
```

Comment:
If file logging is disabled, the title shouldn't say it is writing to the log.
Same with below.

Reply:
>  Also, clicking the X instead of the 'Ok' should not open the log file, users should not have to open it.

Unfortunately, we have no control over this because `alert()` returns `void`. It doesn't return whether the OK button was pressed or whether it was closed through other means.